### DISCUSSION TITLE
关于多个IPV6绑定同一个域名

### DISCUSSION THREAD

**Discussion by cutepengpeng** [jeessy2/ddns-go#1081] [Q&A]
场景：VPS现获分配2001:bec0:165:2a5::/64，同一网卡（eht0）配置多个V6地址（如：2001:bec0:165:2a5::1/64，2001:bec0:165:2a5::2/64，2001:bec0:165:2a5::3/64）
需求：通过DDNS-GO，在CloudFlare上生成多条DNS记录，记录指向的域名可以为相同的二级域名或者不相同的二级域名，请问IDDNS-GO管理页面PV6部分该如何设置？
附折腾过程：
1、通过网卡获取，正向表达式2001:bec0:165:2a5::1[1-9a]，Domains：v1.abc.com、v1.abc.com、v1.abc.com，结果只能生成2001:bec0:165:2a5::a生成v1.abc.com的记录。
2、通过网卡获取，正向表达式流控，Domains：v1.abc.com、v1.abc.com、v1.abc.com，结果只生成一条2001:bec0:165:2a5::a生成v1.abc.com的记录。

希望作者能抽空解答

**Comment by jeessy2** [comment]
首页有说明，参考[传递自定义参数](https://github.com/jeessy2/ddns-go/wiki/%E4%BC%A0%E9%80%92%E8%87%AA%E5%AE%9A%E4%B9%89%E5%8F%82%E6%95%B0)

**Comment by cutepengpeng** [comment]
![微信图片_20240418171647](https://github.com/jeessy2/ddns-go/assets/167413003/0df37724-8240-496f-8266-dfc429ae0476)
不太理解，可否简单写写举例

**Reply by jeessy2** [reply]
那个就是例子。没有更简单的了

**Reply by cutepengpeng** [reply]
可能我原生只是一个学网络的人，对于这种示例理解能力有不足，不过已经解决了并成功了，我希望可以写个示例留给各位像我这样的人

**Reply by jeessy2** [reply]
可以，写个超级详细的文档发来吧

**Reply by cutepengpeng** [reply]
[关于DDNS-go，多个IPV6的绑定的使用方式.docx](https://github.com/jeessy2/ddns-go/files/15032688/DDNS-go.IPV6.docx)
第一次写，比较粗糙望见谅

**Reply by jeessy2** [reply]
能写成markdown的形式不？直接弄到github. 不用下载

**Reply by cutepengpeng** [reply]
能力有限，不会markdown望见谅

**Reply by cutepengpeng** [reply]
刚刚上了速成班，不知道这是否你需要的
https://github.com/cutepengpeng/-DDNS-go-IPV6-/tree/f88dde199b6b6c2505b62e34dc7ab9015abe7e5e/2024-04-19_dtv0v

**Reply by jeessy2** [reply]
可以，内容得优化下，有些错别字。截图可以把域名部分包含，一步一步的进行，可以用假域名。

**Reply by cutepengpeng** [reply]
https://github.com/cutepengpeng/DDNS-GO-IPV6
再次更新

**Reply by jeessy2** [reply]
可以，完善些了。`YourComment`都是可以修改的，这个只有一个唯一号，并不是固定的

**Reply by cutepengpeng** [reply]
再次更新#2

**Reply by jeessy2** [reply]
直接修改要重启，程序读了文件会缓存的，不用没次读。

**Reply by cutepengpeng** [reply]
再次更新#3
增加修改YAML的简单解释
